api/views.py

```diff
@@ -291,7 +291,18 @@ def handle_post(self, request, *args, **kwargs):
         username = body['username']
         password = body['password']
 
-        user = authenticate(username=username, password=password)
+        #allowing different lower/uppercase lettered usernames:
+        caseSensitiveUsername = username
+        try:
+            findUser = User._default_manager.get(username__iexact=username)
```

Comment:
@szabozoltan69 is there a reason why you don't use `User.objects.get()` here? That is the standard way to do this query.